Index: 12.1.51-r0/libspotify-12.1.51-Linux-armv6-release/Makefile
===================================================================
--- 12.1.51-r0.orig/libspotify-12.1.51-Linux-armv6-release/Makefile
+++ 12.1.51-r0/libspotify-12.1.51-Linux-armv6-release/Makefile
@@ -15,7 +15,7 @@ install:
 	mkdir -p $(prefix)/lib/pkgconfig
 	sed -e s:PKG_PREFIX:$(prefix):g <lib/pkgconfig/libspotify.pc >$(prefix)/lib/pkgconfig/libspotify.pc
 
-	ldconfig
+
 
 uninstall:
 	rm -rf $(prefix)/include/libspotify
